src/constants.ts

```diff
@@ -71,12 +71,11 @@ function getConfig(network: string): NetworkConfig {
   return networkConfig
 }
 
-function context(
+export function context(

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums